modules/rn_3_13_0.adoc

```diff
@@ -55,21 +55,26 @@ The following enhancements have been made to the {productname} v2 UI.
 
 The following configuration fields have been added to {productname} {producty}.
 
-[id="oauth-reassign-configuration-field"]
-=== OAuth access token reassignment configuration field 
+[id="disable-pushes-configuration-field"]
+=== Disabling pushes to the {productname} registry configuration field
+
+In some cases, a read-only option for {productname} is not possible since it requires inserting a service key and other manual configuration changes. With the release of {productname} 3.13, a new configuration field has been added: `DISABLE_PUSHES`. 
+
+When `DISABLE_PUSHES` is set to `true`, users are unable to push _new_ images to the registry; however, if the registry already contains the image and a new tag is added, the operation succeeds. All other registry operations continue as normal, for example, changing tags, editing a repository, robot account creation and deletion, user creation, and so on.
+
+This field might be useful in some situations such as when {productname} administrators want to calculate their registry's quota and disable image pushing until after calculation has completed. With this method, administrators can avoid putting putting the whole registry in `read-only` mode, which affects the database, so that most operations can still be done.
 
```

Comment:
Can we also document some words some where for [PROJQUAY-7981](https://issues.redhat.com/browse/PROJQUAY-7981)? let user know,  tag doesn't "Permanently delete" when enable "DISABLE_PUSHES"

Reply:
Done. PTAL!